web/third/bootstrap-typeahead/typeahead.js

```diff
@@ -198,6 +199,11 @@ import {get_string_diff} from "../../src/util";
     return false;
   }
 
+  , new_indicator: function (item_list, items) {
+    // return a string to show in typeahead items or false.
+    return false
+  }
+
```

Comment:
To make this more general and reusable in the future, I think we should name this something like `option_label`. The overriding function(s) should be responsible for specifying the specific label string, but this library function should remain generic. Also, minor correction - the 2nd argument should be singular :)

Reply:
> The overriding function(s) should be responsible for specifying the specific label string

Does this mean that I should move the logic in the override function over to the bootstrap code and just allow the function to just return the label string or is the current changes fine and just needs a name change to `option_label`?

Reply:
The latter, just a name change for the new library function. The related html class would need a similar change.